installer/targets/ros-openpose_ros/openpose_ros.service

```diff
@@ -6,4 +6,4 @@ WantedBy=multi-user.target
 
 [Service]
 Restart=on-abort
-ExecStart=/usr/bin/sudo -H -u ubuntu bash -c 'source /home/ubuntu/.tue/setup.bash && rosrun openpose_ros openpose_ros_node _net_input_width:=368 _net_input_height:=368 _net_output_width:=368 _net_output_height:=368 _model_folder:=/home/ubuntu/openpose/models/'
+ExecStart=/usr/bin/sudo -H -u ubuntu bash -c 'source /home/ubuntu/.tue/setup.bash && rosrun openpose_ros openpose_ros_node _net_input_width:=368 _net_input_height:=368 _net_output_width:=368 _net_output_height:=368 _model_folder:=/home/ubuntu/openpose/models/ __ns:=amigo/openpose'
```

Comment:
Lets test this environment feature separately and align with amigo in a new issue/pr